structure/mysql_vm_repository.go

```
@@ -0,0 +1,184 @@
+func NewMySQLVMRepository(db *sql.DB) *MySQLVMRepository {
+	return &MySQLVMRepository{DB: db}
+}
+
+func (r *MySQLVMRepository) AddInstance(instanceInfo *VMInfo, coreIdx int) error {
+	log := util.GetLogger()
+	tx, err := r.DB.Begin()
+	if err != nil {
+		log.Error("Failed to start transaction %v", err)
+		return err
+	}
+	defer tx.Rollback()
```

golangci-lint (2.11.4)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)